package.json

```diff
@@ -1,30 +1,32 @@
 {
   "name": "gitter-desktop",
   "version": "0.0.1",
-  "nwversion": "0.11.6",
-  "description": "Gitter node-webkit builder",
+  "nwversion": "0.12.3",
+  "description": "Gitter nw.js builder",
   "main": "index.js",
   "dependencies": {
     "gulp": "^3.8.10",
     "gulp-shell": "^0.2.11",
     "gulp-util": "^3.0.1",
     "lodash.template": "^3.0.1",
-    "node-webkit-builder": "^1.0.5",
+    "nw-builder": "^2.2.0",
     "semver": "^4.2.0"
   },
+  "devDependencies": {
+    "bluebird": "^3.2.2",
```

Comment:
@trevorah It is included as a `dependency in the main app: https://github.com/gitterHQ/desktop/pull/100/files#diff-e3600309bfd1f7dbe245b37ce9cd68c2R26